.github/workflows/data-source-integration-test.yml

```diff
@@ -105,6 +105,7 @@ jobs:
             -e TEST_USGS_LANDSAT_USERNAME="${{ secrets.TEST_USGS_LANDSAT_USERNAME }}" \
             -e TEST_USGS_LANDSAT_PASSWORD="${{ secrets.TEST_USGS_LANDSAT_PASSWORD }}" \
             -e PL_API_KEY="${{ secrets.PL_API_KEY }}" \
+            -e CDSAPI_KEY="${{ secrets.CDSAPI_KEY }}" \
```

Comment:
Can you trigger a data source integration workflow run and link it on this pr
?

Reply:
Sure, https://github.com/allenai/rslearn/actions/runs/12286357443

Reply:
alright, the new run passed, I will merge the change.